packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -29,9 +39,18 @@ class DatePicker extends StatefulWidget {
     assert(lastDate != null);
   }
 
+  /// The currently selected date.
+  ///
+  /// This date is highlighted in the picker.
   final DateTime selectedDate;
+
+  /// Called when the user picks a date.
   final ValueChanged<DateTime> onChanged;
+
+  /// The earliest date the user is permitted to pick.
```

Comment:
mention what the default would be? how far back will we go? do we support going back before the gregorian calendar started?